src/test/ui/suggestions/fn-or-tuple-struct-without-args.stderr

```diff
@@ -33,7 +33,7 @@ LL |     let _: usize = foo;
            found fn item `fn(usize, usize) -> usize {foo}`
 help: use parentheses to call this function
    |
-LL |     let _: usize = foo(a, b);
```

Comment:
I could restore these, but I actually don't like them. they're the argument names in the fn, but not actually real valeus in the caller scope. let me know what i should do, though.